### DISCUSSION TITLE
LoTW instead of QSL data in Previous Contacts card

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1806] [Ideas / Feature Requests]
Hi,

When enter QSO data, in Previous Contacts card below map, I would like to see LoTW confirmations instead of QSL confirmations. 

![image](https://user-images.githubusercontent.com/101504692/203841843-3838d2f9-7cd1-40a1-9701-2bbf220927de.png)

Doable? Thanks!

Regards,
Dragan, 4O4A

**Comment by phl0** [comment]
What do you think about this?

![Screenshot from 2022-11-25 00-12-19](https://user-images.githubusercontent.com/7112907/203873176-d5510e26-f29e-4678-8682-7951014f899f.png)

![Screenshot from 2022-11-25 00-12-55](https://user-images.githubusercontent.com/7112907/203873181-614b2784-bd8b-49c4-9725-d16cd7b081f0.png)

**Reply by phl0** [reply]
Made a PR with the changes. See here: https://github.com/magicbug/Cloudlog/pull/1807

**Comment by 4O4A** [comment]
Looks cool, Sky (The Dog) approves it :)
Thank you!

**Reply by phl0** [reply]
Did the sog review the code?
